ci-operator/jobs/janus-idp/backstage-showcase/janus-idp-backstage-showcase-main-periodics.yaml

```diff
@@ -385,3 +385,70 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build09
+  cron: 0 8 * * *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: main
+    org: janus-idp
+    repo: backstage-showcase
+  labels:
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-janus-idp-backstage-showcase-main-e2e-tests-operator-nightly
```

Comment:
```suggestion
  name: periodic-ci-janus-idp-backstage-showcase-main-e2e-tests-operator-nightly
  reporter_config:
    slack:
      channel: '#rhdh-e2e-test-alerts'
      job_states_to_report:
      - success
      - failure
      - error
      report_template: '{{if eq .Status.State "success"}} :done-circle-check: {{else}}<!subteam^S07BMJ56R8S>
        :failed: {{end}} Job *`{{.Spec.Job}}`* ended with *{{.Status.State}}*. Open
        Report: <https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/logs/periodic-ci-janus-idp-backstage-showcase-main-e2e-tests-nightly-aks/{{.Status.BuildID}}/artifacts/e2e-tests-nightly-aks/janus-idp-backstage-showcase-helm-aks-nightly/artifacts/showcase-k8s-ci-nightly/index.html|nightly>
        | <https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/logs/periodic-ci-janus-idp-backstage-showcase-main-e2e-tests-nightly-aks/{{.Status.BuildID}}/artifacts/e2e-tests-nightly-aks/janus-idp-backstage-showcase-helm-aks-nightly/artifacts/showcase-rbac-k8s-ci-nightly/index.html|rbac-nightly>
        | <https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/logs/periodic-ci-janus-idp-backstage-showcase-main-e2e-tests-nightly-aks/{{.Status.BuildID}}/artifacts/e2e-tests-nightly-aks/janus-idp-backstage-showcase-helm-aks-nightly/artifacts/|Open
        artifacts> | <{{.Status.URL}}|View logs>'
```

Manually add this slack configuration for this periodic job alerts. make sure to update the links as per you job names.